doc/nrf/libraries/caf/power_manager.rst

```diff
@@ -15,7 +15,26 @@ Configuration
 
 To enable the |power_manager|, set the :kconfig:option:`CONFIG_CAF_POWER_MANAGER` Kconfig option in the configuration.
 
-This module uses Zephyr's :ref:`zephyr:pm_api` subsystem.
+Implied features
+================
+
+The :kconfig:option:`CONFIG_CAF_POWER_MANAGER` option implies the following features that can be used to reduce power consumption:
+
+* System power off support (:kconfig:option:`CONFIG_POWEROFF`)
+  The option is not implied for nRF54H SoC Series (:kconfig:option:`CONFIG_SOC_SERIES_NRF54HX`), because the :c:func:`sys_poweroff` API is not yet fully supported on nRF54H SoC Series.
+* Device Power Management (:kconfig:option:`CONFIG_PM_DEVICE`)
```

Comment:
```suggestion
* System power off support (:kconfig:option:`CONFIG_POWEROFF`).
  The option is not implied for an nRF54H Series SoC (:kconfig:option:`CONFIG_SOC_SERIES_NRF54HX`), because the :c:func:`sys_poweroff` API is not yet fully supported on the nRF54H Series SoC.
* Device Power Management (:kconfig:option:`CONFIG_PM_DEVICE`).
```